dadi/lib/help.js

```diff
@@ -22,12 +61,10 @@ module.exports.sendBackJSON = function (successCode, res, next) {
 
           break
 
-        case 'BAD_REQUEST':
-          body = {
-            success: false,
-            errors: err.errors,
-            statusCode: 400
-          }
+        case 'UNAUTHORISED':
+          body = Object.assign(formatError.createApiError('0005'), {
+            statusCode: 403
```

Comment:
Mixing unauthorised and forbidden here?

Reply:
You're right. I think in this case, the function should be looking for errors in `req.dadiApiClient`. If it finds one, it means the request did not contain a valid bearer token, so it should return `401`. If it doesn't, it means there is a valid bearer token, just not one with enough permissions to complete the operation, so a `403` is returned.

Does that sound ok?

Reply:
Yes, absolutely 😍